collector/container/konflux.Dockerfile

```diff
@@ -127,4 +127,6 @@ ENTRYPOINT ["collector"]
 LABEL \
     com.redhat.component="rhacs-collector-container" \
     io.k8s.display-name="collector" \
-    name="rhacs-collector-rhel8"
+    name="advanced-cluster-security/rhacs-collector-rhel8" \
+    cpe="cpe:/a:redhat:advanced_cluster_security:4.9::el8"
```

Comment:
[no-op] TIL about [this file](https://gitlab.cee.redhat.com/releng/konflux-release-data/-/blob/main/prodsec/rh-acs.yaml) (directed there from [docs](https://konflux.pages.redhat.com/docs/users/getting-started/passing-ec.html#missing-or-invalid-cpe-label-on-image)).